web_src/js/features/package-setting.ts

```diff
@@ -0,0 +1,6 @@
+import {initCompSearchRepoBox} from './comp/SearchRepoBox.ts';
+
+export function initPackageSettings() {
+  if (!document.querySelector('.page-content.package.settings')) return;
+  initCompSearchRepoBox();
+}
```

Comment:
Why not use our "global event/init" framework. Just like I have shown you in

* #32721
* #33755

Reply:
c0a2fd3072